Replicator/RevFinder.cc

```diff
@@ -220,7 +220,10 @@ namespace litecore::repl {
                 // If the removal flag is accompanyied by the deleted flag, we don't purge, c.f. above remark.
                 auto mode = (deletion < 4) ? RevocationMode::kRevokedAccess
                                            : RevocationMode::kRemovedFromChannel;
-                revoked.emplace_back(new RevToInsert(docID, revID, mode, getCollection()->getSpec(),
+                auto collSpec = getCollection()->getSpec();
+                logInfo("SG revoking: %.*s.%.*s.%.*s/%.*s with mode %u", SPLAT(collSpec.scope), SPLAT(collSpec.name), SPLAT(docID),
```

Comment:
Maybe "SG revoked access to the doc : 'scope.collection.id/revid" or "SG revoked access to the doc 'scope.collection.id' at rev 'revid'"

Reply:
I would like to keep the "mode" to help debugging.